goat.sh

```diff
@@ -173,7 +173,15 @@ handle_input() {
     local exit_status=0
 
     # Test existance of the file with shortcuts
-    [ ! -f "$SHORTCUTS_FILE" ] && touch "$SHORTCUTS_FILE"
+    if [ ! -f "$SHORTCUTS_FILE" ];then 
+	touch "$SHORTCUTS_FILE"
+	create down	~/Downloads &>- 
+	create doc 	~/Documents &>-
+	create root 	/	    &>-     
+	create conf 	~/.config/  &>-
+	create proj 	~/Projects  &>-
+	create home	~	    &>-     	
```

Comment:
I wouldn't add `home` here. `cd` is enough to go home - I do not need to type `cd home` or even `goat home`.